scripts/submit_hf.py

```diff
@@ -44,7 +44,7 @@ def gen_command_template(params, machine, seed=const.HF_DEFAULT_SEED):
     }
     add_args = {}
     for k, v in params.hf.items():
-        if v is False:
+        if v is False or v is None:
```

Comment:
`if not v` ? Seems like that would be a little more complete?

Reply:
But if v is 0 like for the seed, then it'll be a false positive and not provided seed 0